jobs/kubectl/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG BASE_IMAGE
 
-FROM bitnami/kubectl:1.29.6 as kubectl
+FROM bitnami/kubectl:1.31.1 as kubectl
```

Comment:
can you also modify `jobs/kubectl/Dockerfile.distroless`?

Reply:
Great catch! Done